src/Helpers/Str.php

```diff
@@ -38,48 +41,6 @@ public static function makeSafe($str, $strict = false, $extrachars = '')
     }
 
     /**
-     * Replace the first occurrence of a string only. Behaves like str_replace, but
-     * replaces _only_ the _first_ occurrence.
-     *
-     * @see http://stackoverflow.com/a/2606638
-     *
-     * @param string $search
-     * @param string $replace
-     * @param string $subject
-     *
-     * @return string
-     */
-    public static function replaceFirst($search, $replace, $subject)
-    {
-        $pos = strpos($subject, $search);
-        if ($pos !== false) {
-            $subject = substr_replace($subject, $replace, $pos, strlen($search));
-        }
-
-        return $subject;
-    }
-
-    /**
-     * Replace only the last occurrence of the $search text on the $subject.
-     *
-     * @param string $search
-     * @param string $replace
-     * @param string $subject
-     * @param bool   $caseSensitive
-     *
-     * @return string
-     */
-    public static function replaceLast($search, $replace, $subject, $caseSensitive = true)
-    {
-        $pos = $caseSensitive ? strrpos($subject, $search) : strripos($subject, $search);
-        if ($pos !== false) {
-            $subject = substr_replace($subject, $replace, $pos, strlen($search));
-        }
-
-        return $subject;
-    }
```

Comment:
These are BC breaks. The methods should be kept here and flip the parameters before calling parent.